config_defaults/subtests/docker_cli/login.ini

```diff
@@ -0,0 +1,12 @@
+[docker_cli/login]
+subsubtests = login_ok,login_fail,logout_ok,push_ok,push_fail
+#: docker image containing a functional registry, including auth checks.
+#: Make sure to include this in config_custom/defaults.ini:preserve_fqins !
```

Comment:
You could just add a ``preserve_fqins`` here and override the default...errr...wait, nope that won't work.  Never-mind.